### ISSUE TITLE
User not permitted to edit submission

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#405]
**Description**
Unfortunately this issue from the other day seems to not be completely resolved, while the user can now see the save button, they get the following error on trying to save when they only have manage permissions.

https://github.com/verbb/formie/issues/396

![Screen Shot 2021-03-19 at 10 31 10 am](https://user-images.githubusercontent.com/21274925/111712819-565bd000-889e-11eb-9059-e6a332f5fa1a.png)

**Additional info**
- Plugin version: 1.3.24
- Craft version: 3.6.10

**Comment by engram-design** [comment]
Ah, sorry about that. Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.24",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Great! Thank you for the quick fix

**Comment by engram-design** [comment]
Should be fixed in [1.3.25](https://github.com/verbb/formie/releases/tag/1.3.25)
